components/Post.js

```diff
@@ -119,6 +116,8 @@ export const Post = (post) => {
 		}
 	};
 	const tS = timeSince(post.timestamp);
+	const titleCharThreshold = 21;
```

Comment:
After extensive testing with one device, I suggest we make this based on the screen width (minus the x padding, the Profile pic width, and the post type icon). Implementing a dynamic post title width will allow for a standard title length before truncation.